scripts/installdeps.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 set -eux
+pushd "$(dirname ${BASH_SOURCE:0})/.."
+trap popd EXIT
```

Comment:
Great idea!
Since subshells cannot change the parent, we can skip pushd/popd and just cd to the required directory.